pkg/config/node.go

```diff
@@ -36,9 +39,11 @@ type Cluster struct {
 }
 
 type Backend struct {
-	Host    string
-	Address string
-	Port    uint16
+	Host               string
+	CanonicalHost      string
+	CanonicalShortName string
```

Comment:
Do we use CanonicalHost and CanonicalShortName ?

Reply:
Yes. It's being used in the templates.

Reply:
I "greped" MCO and couldn't find these two fields. what am I missing here?